src/overdub.jl

```diff
@@ -132,9 +132,16 @@ function overdub_pass!(reflection::Reflection,
     # this hack's first assumption: that `Core.kwfunc(f)` is going to return a function whose
     # type name is prefixed by `#kw##`. More assumptions for this hack will be commented on
     # as we go.
-    iskwfunc = startswith(String(signature.parameters[1].name.name), "#kw##")
+    name = String(signature.parameters[1].name.name)
+    iskwfunc = startswith(name, "#kw##")
     istaggingenabled = hastagging(context_type)
 
+    is_calloverload = false
+    if length(code_info.slotnames) > 0
+        # TODO: is this sufficient?
+        is_calloverload = code_info.slotnames !== Symbol("#self#")
```

Comment:
That doesn't look right.
I think you mean to be checking the first element of that vector,not comparing the whole vector to a scalar

Reply:
Indeed.